### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by dikshadhote**
Preview grid : https://famous-puffpuff-5ea85c.netlify.app/documents/documents.html#grid
Features added 
1. Two column grid
2. Three Column grid 

PS: Code formatter is not working in VS code. Please bear with unstructured code lines

image

image

### CODE REVIEW

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+.col-2-stretch {
+  display: grid;
+  grid-template-columns: 70% 30%;
+  grid-column-gap: 5px;
+}
+
+.col-3 {
+  display: grid;
+  grid-template-columns: 2fr 2fr 2fr;
```
Comment (JantuDeb) [inline components/grid.css:21]: You can use `repeat` here

File: components/grid.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.col-2 {
+  display: grid;
+  grid-template-columns: 50% 50%;
```
Comment (ivinayakg) [inline components/grid.css]: dont need to give percent just give 1fr 1fr or there is some other reason for this?

**Review by JantuDeb — APPROVED** [review]
Code looks structured and clean

**Review by ivinayakg — APPROVED** [review]
I like the fact that you created multiple variants for the grid component which were not even required, Great work 👍!

File: components/grid.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.col-3 {
+  display: grid;
+  grid-template-columns: 2fr 2fr 2fr;
+  grid-column-gap: 5px;
+}
+
+.col-3-left-expand {
+  display: grid;
+  grid-template-columns: 3fr 1.5fr 1.5fr;
```
Comment (harshalini) [inline components/grid.css:27]: I am not sure, but I think you can use here 2fr 1fr 1fr as fr only divides in equal parts. Code is understandable!

**Review by harshalini — APPROVED** [review]

**Pull request merged**